scheds/rust/scx_layered/src/main.rs

```diff
@@ -2282,6 +2298,28 @@ impl<'a> Scheduler<'a> {
         Ok(())
     }
 
+    fn refresh_idle_qos(&mut self) -> Result<()> {
+        if !self.idle_qos_enabled {
+            return Ok(());
+        }
+
+        let mut cpu_idle_qos: HashMap<usize, i32> =
+            (0..*NR_CPUS_POSSIBLE).map(|cpu| (cpu, 0)).collect();
```

Comment:
We do the same thing elsewhere too but this should be `NR_CPU_IDS`. Also, wouldn't a vector make more sense?